youtrack_sdk/client.py

```diff
@@ -164,7 +164,7 @@ def create_issue(self, *, issue: Issue) -> Issue:
             ),
         )
 
-    def update_issue(self, *, issue_id: str, issue: Issue, mute_update_notifications: bool) -> Issue:
+    def update_issue(self, *, issue_id: str, issue: Issue, mute_update_notifications: Optional[bool] = None) -> Issue:
```

Comment:
So I wonder what happens if it's `None` and this method is called? Is the payload `mute_update_notifications` sent with the value of `null`? I would find this unfortunate. Who knows how YouTrack understands the difference between `null` and `undefined` - unfortunately not everybody follows Zalando guidelines.

Also, I don't think that sending the field with the value `False` is problematic for users that don't have permission to call it with `True`. Instead of making it optional, which for me is a rather questionable semantic, I would have set it to `False` per default and that's it.

If the user then sets it to `True` and doesn't have required permissions, then a permission error will come.

Reply:
if `mute_update_notifications` is `None` it is not added as part of the payload

https://github.com/moneymeets/youtrack-sdk/blob/90b3f66a2572109117652e4075c5cd0051c92863/youtrack_sdk/client.py#L54

Reply:
> So I wonder what happens if it's `None` and this method is called? Is the payload `mute_update_notifications` sent with the value of `null`? I would find this unfortunate. Who knows how YouTrack understands the difference between `null` and `undefined` - unfortunately not everybody follows Zalando guidelines.

Fields with `None` value are excluded from the payload.

> Also, I don't think that sending the field with the value False is problematic for users that don't have permission to call it with True. Instead of making it optional, which for me is a rather questionable semantic, I would have set it to False per default and that's it.

If it works for `False` then I agree with you. Need to be tested @IamAbbey

Reply:
I think that @IamAbbey doesn't have the rights to apply changes silently - the button should be missing from the command box. If that's correct, he can easily test that.